vrp-cli/src/lib.rs

```diff
@@ -328,6 +329,7 @@ mod c_interop {
                 problem.as_ptr() as *const c_char,
                 matrices.as_ptr() as *const *const c_char,
                 0,
+                false,
```

Comment:
That is also an compilation error: config is passed before geojson flag. Please run `cargo test` to catch compilation errors.